bootstrap/sql/postgresql/v005_add_reconfigure_flag.sql

```diff
@@ -0,0 +1,24 @@
+-- Copyright 2017 Hortonworks.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- Add reconfigure flag to components
+ALTER TABLE "topology_test_run_case_source" ADD COLUMN "sleepMsPerIteration" BIGINT NOT NULL DEFAULT 0;
```

Comment:
This is covered by v003.
https://github.com/hortonworks/streamline/blob/master/bootstrap/sql/postgresql/v003__add_sleep_iteration_to_topology_test_run_case_source.sql

Looks like missing to remove the line after copying.